test_settings.py

```diff
@@ -73,3 +57,68 @@
     SOUTH_MIGRATION_MODULES = {
         'waffle': 'waffle.south_migrations'
     }
+
+if django.VERSION < (1,8):
+    # Old style TEMPLATE_* settings.
+    # Use jingo.
+
+    TEMPLATE_LOADERS = (
+        'jingo.Loader',
+        'django.template.loaders.filesystem.Loader',
+        'django.template.loaders.app_directories.Loader',
+    )
+
+    TEMPLATE_CONTEXT_PROCESSORS = (
+        'django.contrib.auth.context_processors.auth',
+        'django.core.context_processors.request',
+    )
+
+    JINGO_EXCLUDE_APPS = (
+        'django',
+        'waffle',
+    )
+
+else:
+    # New style TEMPLATES setting.
+    # Use django-jinja
+
+    INSTALLED_APPS += ('django_jinja', )
+
+    TEMPLATES = [
+        # Configure the Jinja2 template engine
+        {
+            'BACKEND': 'django_jinja.backend.Jinja2',
+            'DIRS': [],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                "match_regex": r"jingo.*",
+                'match_extension': '',
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'django.contrib.messages.context_processors.messages',
```

Comment:
I don't think we need the `debug` and `messages` context processors. I'm not sure it makes much of a difference, but I think I would leave them out to match the jingo configuration.